### DISCUSSION TITLE
Getting 500 for all apis suddenly with response as `could not write init file`

### DISCUSSION THREAD

**Discussion by PrasanthKumar77** [ToolJet/ToolJet#4375] [Q&A]
i am using 1.14.0 version. we are using the tooljet since a month and haven't made any changes to it recently. but from today apps are not being loaded. when i  checked the apis in browser they are returning 500 status cod with following response. we have freezed the version in docker file.
```
{
    "statusCode": 500,
    "timestamp": "2022-10-11T12:54:42.425Z",
    "path": "/api/folders?searchKey=",
    "message": "could not write init file"
}
```

when logged out getting this 
```
{
    "statusCode": 500,
    "timestamp": "2022-10-11T13:00:56.616Z",
    "path": "/api/organizations/public-configs",
    "message": "getaddrinfo ENOTFOUND postgres"
}
```

**Comment by akshaysasidrn** [comment]
Hello @PrasanthKumar77 , could you share more information regarding how your setup is implemented? From the error logs that are shared, it looks like something to do with the database. Is it running properly? Are you able to connect to it separately and execute any query? From a quick search, I infer that `could not write init file` would be logged when the hard drive is full.
